.travis.yml

```diff
@@ -5,7 +5,7 @@ dist: trusty
 
 env:
   global:
-  - R_CHECK_ARGS="--no-build-vignettes --no-manual --timings"
+  - R_CHECK_ARGS="--no-build-vignettes --no-manual --timings --run-donttest"
```

Comment:
why 'run-donttest'?
wont we miss new bugs?

Reply:
A lot of examples are commented in `donttest`. `--run--donttest` will make the travis check these examples.

Reply:
Although i see `found \donttest examples: check also with --run-donttest` in travis so this might not work.

Reply:
Thanks for clarification!